src/hotspot/cpu/x86/vm_version_x86.cpp

```diff
@@ -956,6 +956,7 @@ void VM_Version::get_processor_features() {
   if (UseAVX < 1) {
     _features &= ~CPU_AVX;
     _features &= ~CPU_VZEROUPPER;
+    _features &= ~CPU_F16C;
```

Comment:
Is `is_knights_family()` supports `f16c`?  We switch off some avx512 features for it. But it looks like `f16c` is not connected to `avx512`.

Reply:
Hi Vladimir, you're correct that f16c is not connected to avx512.